services/galley/src/Galley/API.hs

```diff
@@ -133,6 +133,7 @@ sitemap = do
   get "/teams/:tid/members" (continue getTeamMembers) $
     zauthUserId
       .&. capture "tid"
+      .&. def (unsafeRange 2000) (query "maxResults")
```

Comment:
1. don't forget the swagger docs!
2. what will the error look like to the client if it tries to set this to 3000?

Reply:
1. I added the swagger docs.
1. If a client tries to set it to 3000, it is rejected with status code 400 and this in error message:
```
[query] 'maxResults' invalid: Failed reading: outside range [1, 2000]
```